### PR TITLE
Get UserAgent from package.json

### PR DESCRIPTION
**Pull request by AndrewFerr**
Fixes https://github.com/matrix-org/matrix-hookshot/issues/359

Signed-off-by: Andrew Ferrazzutti <andrewf@element.io>

### CODE REVIEW

**Review by tadzik — APPROVED** [review]
Looks good from here!

You'll need a changelog entry too, see existing ones in `changelog.d` for inspiration.

File: changelog.d/359.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Get UserAgent from package.json
```
Comment (tadzik) [inline changelog.d/359.bugfix:1]:
```suggestion
Match UserAgent version to Hookshot's version
```

To be a little more precise? As-is you may think that the whole useragent is configurable in package.json :)

**Comment by AndrewFerr** [comment]
4821438 and 97c5d79 update the changelog file to follow the proposed [contribution guide](https://github.com/matrix-org/matrix-appservice-bridge/pull/404).

I've also put the `Signed-off-by` trailer in the PR comment, as I had forgotten to signoff previous commits & am opting to use the PR comment to cover all of the commits, if that's preferable to amending the commits.

**Comment by tadzik** [comment]
Still looks good. Feel free to merge when ready :)

**Pull request merged**